stories/dashboard.stories.js

```diff
@@ -24,6 +24,7 @@ import PostSearcher from '../assets/js/components/PostSearcher';
 import { googlesitekit as analyticsDashboardData } from '../.storybook/data/wp-admin-admin.php-page=googlesitekit-module-analytics-googlesitekit';
 import { STORE_NAME as CORE_SITE } from '../assets/js/googlesitekit/datastore/site/constants';
 import { STORE_NAME as ANALYTICS_STORE } from '../assets/js/modules/analytics/datastore/constants';
+import { STORE_NAME as SEARCH_CONSOLE_STORE } from '../assets/js/modules/search-console/datastore/constants';
```

Comment:
@ryanwelcher looks like there was some inconsistency before, but would you please update these to use the same naming convention we use elsewhere for the store name alias?
`CORE_SITE` is correct above, but `ANALYTICS_STORE` should be `MODULES_ANALYTICS` and `SEARCH_CONSOLE_STORE` should be `MODULES_SEARCH_CONSOLE`.